editor/components/post-publish-panel/toggle.js

```diff
@@ -33,17 +46,31 @@ function PostPublishPanelToggle( { isSaving, isPublishable, isSaveable, isPublis
 			disabled={ ! isButtonEnabled }
 			isBusy={ isSaving && isPublished }
 		>
-			<PostPublishButtonLabel />
-			<Dashicon icon="arrow-down" />
+			{ __( 'Publish...' ) }
```

Comment:
Poor man's ellipsis: `...` (not semantically meaningful)
Rich man's ellipsis: `…` (semantically meaningful)